hypothesis-python/tests/ghostwriter/test_ghostwriter_cli.py

```diff
@@ -116,6 +121,162 @@ def test_can_import_from_scripts_in_working_dir(tmpdir):
     assert "Error: " not in result.stderr
 
 
+CLASS_CODE_TO_TEST = """
+from typing import Sequence, List
+
+def func_sorter(seq: Sequence[int]) -> List[int]:
+    return sorted(seq)
+
+class S:
```

Comment:
I'd find this easier to read with a module named `my_module` and a class named `MyClass`.